siliconcompiler/checklists/oh_tapeout/oh_tapeout.py

```diff
@@ -1,35 +1,40 @@
 import siliconcompiler
 
+from siliconcompiler import Checklist
+
 def make_docs():
     '''Subset of OH! library tapeout checklist.
 
     https://github.com/aolofsson/oh/blob/master/docs/tapeout_checklist.md
     '''
     chip = siliconcompiler.Chip()
-    setup(chip)
-    return chip
+    return setup(chip)
 
 def setup(chip):
     standard = 'oh_tapeout'
 
+    checklist = Checklist(chip, standard)
```

Comment:
Does `siliconcompiler.Checklist` work without the `from siliconcompiler import Checklist` from above? Ditto for Flow/PDK/Library modules.

Reply:
it should, I can update them to do `siliconcompiler.???`

Reply:
Up to you - it's one less line, but the only important change here is the `_readonly` attribute.